packages/s3-credentials-endpoint/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@cumulus/api": "8.1.0",
     "@cumulus/aws-client": "8.1.0",
+    "@cumulus/cmrjs": "8.1.0",
```

Comment:
9.0.0?

Reply:
v9.0.0 isn't released yet.  If this beats it to master, it'll stay 8.1.0, or I'll have to updated it if it wins the race.